docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md

```diff
@@ -20,8 +20,9 @@ This article explains how to install the Microsoft ODBC Driver for SQL Server on
 
 This article provides commands for installing the ODBC driver from the bash shell. If you want to download the packages directly, see [Download ODBC Driver for SQL Server](../download-odbc-driver-for-sql-server.md).
 
-> [!Note]
-> The Microsoft ODBC driver for SQL Server on macOS is only supported on the x64 architecture through version 17.7. Apple M1 (ARM64) support was added starting with version 17.8. The architecture will be detected and the correct package will be automatically installed by the Homebrew formula. If your command prompt is running in x64 emulation mode on the M1, the x64 package will be installed. If you're not running in emulation mode in your command prompt, the ARM64 package will be installed.
+> :warning: Note
+> The Microsoft ODBC driver for SQL Server on macOS supports the x64 architecture through version 17.7. In version 17.8, Apple Silicon (ARM64) support was added, and will be the only architecture to receive future support. The architecture will be detected and the correct package will be automatically installed by the Homebrew formula. If your terminal is running in x64 emulation mode on the M1, the x64 package will be installed. If you're not running in emulation mode in your terminal, the ARM64 package will be installed. 
```

Comment:
> In version 17.8, Apple Silicon (ARM64) support was added, and will be the only architecture to receive future support.

This is incorrect. Both architectures are currently supported and no decision has been made to stop ODBC driver support for x64.